### PR TITLE
Add Mocked Tests: Image Actions

### PR DESCRIPTION
**Pull request by loosla**
This PR is adding mocked tests for image_actions.get, image_actions.list, image_actions.post

Related to https://github.com/digitalocean/pydo/issues/211

### CODE REVIEW

File: tests/mocked/test_image_actions.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    }
+
+    responses.add(
+        responses.GET,
+        f"{mock_client_url}/v2/images/{image_id}/actions/{action_id}",
+        json=expected,
+    )
+
+    get_resp = mock_client.image_actions.get(image_id=image_id, action_id=action_id)
+    # TODO (to remove):
```
Comment (loosla) [inline tests/mocked/test_image_actions.py:42]: @loosla remove TODOs in file.

File: tests/mocked/test_image_actions.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    # 1. action_id is missing in openapi spec for Python in Request samples:
+    # https://docs.digitalocean.com/reference/api/api-reference/#operation/imageActions_get
+    # 2. image_id in Python Request samples is better be updated
+    # with the value in response - 7938269 - to avoid confusion
+    # Should we update openapi spec? Should the ticket be created for this update?
```
Comment (andrewsomething) [inline tests/mocked/test_image_actions.py:47]: Great catches! We should get those fixed up.

**Comment by andrewsomething** [comment]
> We may want to fix Makefile: PYLINT_ARGS is empty for make lint-tests.
> Not sure what the intention is: should we remove PYLINT_ARGS or assign the same value as in github actions -  https://github.com/digitalocean/pydo/blob/main/.github/workflows/pr.yaml#L40

We set `PYLINT_ARGS="--disable=fixme"` in the GitHub workflow so that `TODO` comments do not blocker pull requests as we have a number of them in the code base. When run locally, it may be useful to quickly find where we have `TODO` comments that still need to be addressed. We pass the empty `PYLINT_ARGS` in the Makefile so that it is passed through in the environment. If you want to disable the warnings locally, you can run:

```
PYLINT_ARGS="--disable=fixme" make lint-tests
```

File: tests/mocked/test_image_actions.py (hunk 2 above)
Comment (loosla) [reply tests/mocked/test_image_actions.py:47]: Done in https://github.com/digitalocean/openapi/pull/910

**Review by andrewsomething — APPROVED** [review]
:+1:  LGTM!

**Pull request merged**